indico/modules/events/editing/operations_test.py

```diff
@@ -41,6 +46,7 @@ def test_cannot_undo_review_old_rev(dummy_contribution, dummy_user):
 def test_can_undo_review(db, dummy_contribution, dummy_user, type, ok):
     from indico.modules.events.editing.operations import InvalidEditableState, undo_review
     editable = Editable(contribution=dummy_contribution, type=EditableType.paper)
+    EditingRevision(editable=editable, user=dummy_user, type=RevisionType.ready_for_review)
```

Comment:
Why is this extra revision needed?

Reply:
Because I'm logging the editable state transition that happened when undoing the revision. If the editable has only one revision and it's undone, the new state will be null and and the logging fails.
Besides, it should never be possible to have an editable with zero valid revisions. Should I add extra checks/tests to ensure that? (since the first revision is always `new` or `ready_for_review`, it's not possible to undo them anyway, but could help make the logic clearer)